templates/header.html.ep

```diff
@@ -4,8 +4,12 @@
       <ul class="pure-menu-list">
         <li class="pure-menu-item"><%= link_to 'Home', 'landing_page', class => 'pure-menu-link' %></li>
         <li class="pure-menu-item"><%= link_to 'Travel', 'conference.page', {page => 'travel'}, class => 'pure-menu-link' %></li>
+      % if (session 'uid') {
+        <li class="pure-menu-item"><%= link_to 'Profile', 'user.profile', class => 'pure-menu-link' %></li>
+      % } else {
+        <li class="pure-menu-item"><%= link_to 'Login', 'user.profile', class => 'pure-menu-link' %></li>
+      % }
```

Comment:
Yeah, I did consider that, but I'm fairly certain the links will change soon. I also think it's more explicit to read the markup this way.